<launch>
	<!-- declare arg to be passed in -->
	<arg name="ns" default="/"/>
	<arg name="top_map" default="fork_map"/>

	<node pkg="rasberry_des" type="pickers_only.py" name="pickers_only" args="$(arg ns) $(arg top_map)" />

</launch>
